static/css/devhub/new-landing/sections/connect.less

```diff
@@ -0,0 +1,159 @@
+.DevHub-Connect{
+    h2 {
+        font-size: 24px;
+        @media @large {
+          font-size: 36px;
+        }
+    }
+    font-size: 14px;
+}
+
+.DevHub-content-container--Connect {
+    flex-direction: column;
+}
+
+.DevHub-content-header {
+    width: 100%
+}
+
+.Devhub-content-copy--Connect {
+    p, li {
+        color: @color-scheme-dark-items;
+    }
+    ul {
+        list-style-type: none;
+        padding: 0;
+    }
+    li {
+        line-height: 24px;
+        a {
+            .link(@color-scheme-dark-text);
+        }
+        margin-bottom: 20px;
+    }
+    ul.DevHub-content-copy--Connect-twitter-list li a {
+        display: block;
+    }
+
+    @media @medium {
+        display: flex;
+        max-width: 100%
+    }
+}
+
+.DevHub-Connect-section {
+    h4 {
+        margin: 10px 0;
+    }
+
+    @media @medium {
+        width: 33%;
+    }
+}
+
+.DevHub-Connect-newsletter-form {
+    display: flex;
+    flex-flow: row wrap;
+
+    label {
+        color: @color-scheme-dark-items;
+        font-size: 12px;
+    }
+
+    .form_group {
+        display: block;
+        margin-bottom: 20px;
+    }
+
+    .form_group-agree {
+        padding-left: 1.5em; // roughly width of a checkbox
+        padding-right: 0;
+        text-indent: -1.5em;
+
+        html[dir=rtl] & {
+            padding-left: 0;
+            padding-right: 1.5em;
+        }
+    }
+
+    .form_input {
+        background: rgba(177,177,179,0.2);
+        color: #fff;
+        border: 0;
+        padding: 6px;
+        line-height: 20px;
+        width: 100%;
+    }
+
+    #newsletter_privacy {
+        order: 2;
+        a {
+            .link(@color-scheme-dark-text, @decoration: underline);
+        }
+        width: 100%;
+    }
+
+    #newsletter_email {
+        flex-basis: 100px;
+        min-width: 100px;
+        flex-grow: 1;
+
+        label {
+            display: none;
+        }
+    }
+
+    #newsletter_submit {
+        flex-basis: 90px;
+        padding: 0 10px;
+
+        button {
+            height: 32px;
+            background-color: @color-scheme-dark-text;
+            color: @color-scheme-dark-background;
+            font-size: 14px;
+        }
+    }
+
+    #newsletter_errors {
+        display: none; /* hide by default */
+        margin-bottom: 20px;
+        box-shadow: 0 0 0 1px #ffffff; // incase it's on a black background
+        padding: 20px;
+        background-color: rgba(0, 0, 0, 0.1); // incase it's on a white background
+        border: 1px solid #c00;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+}
+
+
+.newsletter_thanks {
+    display: none;
+}
+
+a.Before-Icon::before {
+    padding-right: 0.3em;
```

Comment:
I think we need `rtl` support here too:

![Screen Shot 2019-10-22 at 11 19 31](https://user-images.githubusercontent.com/217628/67273141-2ac49280-f4be-11e9-8274-2e34c0639b76.png)
![Screen Shot 2019-10-22 at 11 19 34](https://user-images.githubusercontent.com/217628/67273142-2b5d2900-f4be-11e9-849c-c08f31358940.png)

Reply:
https://github.com/mozilla/addons-server/pull/12669/commits/d4524db8833c0073fca31723c3007479af69e7ce